packages/dappmanager/src/calls/wifi.ts

```diff
@@ -0,0 +1,92 @@
+import { ComposeFileEditor } from "../modules/compose/editor";
+import { logContainer } from "../modules/docker/api";
+import { listContainer } from "../modules/docker/list";
+import { CurrentWifiCredentials, WifiReport } from "../types";
+import params from "../params";
+
+const WIFI_KEY_SSID = "SSID";
+const WIFI_KEY_PASSWORD = "WPA_PASSPHRASE";
+
+/**
+ * Return wifi report
+ */
+export async function wifiReportGet(): Promise<WifiReport> {
+  const wifiContainer = await listContainer({
+    containerName: params.wifiContainerName
+  });
+  if (!wifiContainer)
+    throw Error(
+      `Wifi is not present. Container name: ${params.wifiContainerName}`
```

Comment:
What does it mean "Wifi is not present"? If a container is not in the list means that there is no container with that name. In DAppNode terminology it means that it's not installed. Think if you were a user and find yourself in that situation, what are the next steps?